src/plugins/intel_cpu/src/plugin.h

```diff
@@ -46,6 +46,19 @@ class Engine : public InferenceEngine::IInferencePlugin {
 
     void ApplyPerformanceHints(std::map<std::string, std::string> &config, const std::shared_ptr<ngraph::Function>& ngraphFunc) const;
 
+    struct StreamCfg {
+        int num_streams;
+        int big_core_streams;          // Number of streams in Performance-core(big core)
+        int small_core_streams;        // Number of streams in Efficient-core(small core)
+        int threads_per_stream_big;    // Threads per stream in big cores
+        int threads_per_stream_small;  // Threads per stream in small cores
+        int small_core_offset;
```

Comment:
From my undesranding "small_core_offset" is part of MappingInfo. In other words I expect StreamExecutor to handle it implicitly from the plugin perspective,

Reply:
"core offset" will be remove in following PRs after we implement searching resource in CPU mapping table instead of calculating resource based on core offset.